microsite/pages/en/plugins.js

```diff
@@ -43,41 +49,234 @@ const Plugins = () => (
         </span>
       </div>
       <BulletLine style={{ width: '100% ' }} />
+      <div className="PluginPageHeader">
+        <h2>Core Features</h2>
+      </div>
       <Container wrapped className="PluginGrid">
-        {pluginMetadata.map(
```

Comment:
Can we not do the inverse of the below rather than repeating?

```ts
{pluginMetadata
          .filter(plugin => plugin.category === 'Core Feature')
          .map(
            ({
              iconUrl,
              title,
              description,
              author,
              authorUrl,
              documentation,
              category,
            }) => (
```

That should remove a lot of code?

Reply:
Hi @benjdlambert , I initially did that but it's displayed in alphabetical order with Kubernetes coming first.

As @fengypants wants to maintain the order (like in this link https://github.com/backstage/backstage/issues/4897#issue-826503762), I've implemented it according to what he suggested here https://github.com/backstage/backstage/issues/4897#issuecomment-802087458.

What do you think?

Reply:
Ok in that case maybe add an optional `order` property to the plugins data. And use that the sort the array.

It's nicer than repeating code blocks.

Reply:
@benjdlambert I've made the changes you've mentioned. Please have a look at it. 😄